src/stuf/stuf_zds/client.py

```diff
@@ -240,6 +241,11 @@ def create_zaak(
                 "zds_zaaktype_status_omschrijving"
             ),
             "zaak_omschrijving": self.zds_options["omschrijving"],
+            "co_signer": (
+                self.zds_options.get("co_sign_data", {})["value"]
+                if self.zds_options.get("co_sign_data")
+                else {}
+            ),
```

Comment:
```suggestion
            "co_signer": (
                co_sign_data["value"]
                if (co_sign_data := self.zds_options.get("co_sign_data"))
                else {}
            ),
```
the `.get` with a default value is confusing if you do the check that it's not empty in the first place